front/components/data_source_view/DataSourceViewSelector.tsx

```diff
@@ -97,7 +96,7 @@ function VaultSelector({
   } else {
     return (
       <div>
-        <Page.Separator />
+        <div className="h-px w-full bg-structure-200" />
```

Comment:
Why do we need a 1 pixel height here 🤔 (`h-px`)

Reply:
The figma component has one pixel and to avoid removing from other components above and below depending on the case.